Description: Do not run configure at the end of autogen.sh
Auhtor: Ludovico cavedon <cavedon@debian.org>

Index: ndpi/autogen.sh
===================================================================
--- ndpi.orig/autogen.sh
+++ ndpi/autogen.sh
@@ -35,4 +35,4 @@ fi
 
 cat configure.seed | sed "s/@NDPI_MAJOR@/$NDPI_MAJOR/g" | sed "s/@NDPI_MINOR@/$NDPI_MINOR/g" | sed "s/@NDPI_PATCH@/$NDPI_PATCH/g" | sed "s/@NDPI_VERSION_SHORT@/$NDPI_VERSION_SHORT/g" > configure.ac
 autoreconf -ivf
-./configure $*
+#./configure $*
